Makefile

```diff
@@ -14,7 +16,10 @@ install:
 install-dependencies:
 	go get -u github.com/stretchr/testify/...
 install-tools:
+
 test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
Well now I see the "-v" flag. Ok, remove the -race argument since I presume this -verbose has another purpose. 
Incorporate both changes, then you may ships it.